packages/web/src/components/edit/fields/AdvancedField.tsx

```diff
@@ -164,11 +167,14 @@ const AdvancedFormSchema = z
 
 export type AdvancedFormValues = z.input<typeof AdvancedFormSchema>
 
-export const AdvancedField = () => {
+export type AdvancedFieldProps = { isHidden?: boolean }
+export const AdvancedField = ({ isHidden }: AdvancedFieldProps) => {
```

Comment:
if you want to get `isHidden` with `useField` here, we have an example in https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/components/edit/fields/visibility/VisibilityField.tsx#L71